README.md

```
@@ -10,6 +10,274 @@
+
+Zircon requires the following dependencies to build and run Zirco:
+
+- **Rust** (install from [rustup.rs](https://rustup.rs/))
+- **LLVM 20** (**REQUIRED** - Zirco only works with LLVM 20.x)
+- **clang** (usually included with LLVM)
+- **Git**
+
+#### Installing LLVM 20 and clang
+
+**On macOS (Homebrew):**
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)